[Woo POS][App Listings] POS ASC screenshots 2

iamgabrielma opened the pull request:
tests will fail depending on the order: If triggered after products tests, we lock screen for notifications. If triggered before, we need to exit POS mode to continue the rest of tests

<!-- Remember about a good descriptive title. -->

<!-- Id number of the Linear issue this PR addresses. -->
<!-- Part of: # -> use this if your PR is one of many related to one issue -->

## Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->

## Test Steps
<!-- Describe how to test your changes. Include only what’s needed for the reviewer to validate the behavior.
For new features, outline the main user flow or goal rather than every tap or click — the reviewer should be able to complete the flow naturally.
If applicable, mention key devices, scenarios, or edge cases to verify. -->

## Screenshots
<!-- Include before and after images or gifs when appropriate. -->

---
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

iamgabrielma committed:
Adjust test order and exit pos accessibility id

tests will fail depending on the order: If triggered after products tests, we lock screen for notifications. If triggered before, we need to exit POS mode to continue the rest of tests

dangermattic commented:
<!--
  0 Errors
  1 Warning: View files have been modified,...
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

AliSoftware committed:
Add `en-GB` to list of screenshot locales

 - Rename the constants for clarity and consistency with other related ones
   - `IOS_LOCALES` -> `SCREENSHOTS_LOCALES`
   - `SCREENSHOT_DEVICES` -> `SCREENSHOTS_DEVICES`
 - Add `en-GB` to `SCREENSHOTS_LOCALES`
 - Improve error messages in case of invalid/unsupported locale or device provided as parameter of the lane

dangermattic commented on fastlane/Fastfile:
```diff
@@ -882,15 +882,17 @@ platform :ios do
-    devices = SCREENSHOT_DEVICES
+    languages = SCREENSHOTS_LOCALES
+    languages &= options[:languages].split(',')
+    if languages.empty?
```
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/e6de4e9a1ee1f97cce171fbd50947844fbbc7e1b/fastlane/Fastfile#L887"></span>Style/IfUnlessModifier: Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>

dangermattic commented on fastlane/Fastfile:
```diff
@@ -882,15 +882,17 @@ platform :ios do
     devices &= options[:devices].split(',') unless options[:devices].nil?
-
-    UI.user_error!("Unable to run on devices: \"#{devices}\"") if devices.empty?
+    if devices.empty?
```
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/e6de4e9a1ee1f97cce171fbd50947844fbbc7e1b/fastlane/Fastfile#L893"></span>Style/IfUnlessModifier: Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>
